release_bot/releasebot.py

```diff
@@ -261,8 +271,12 @@ def run(self):
                 try:
                     if self.new_release.get('trigger_on_issue') and self.find_open_release_issues():
                         if self.new_release.get('labels') is not None:
-                            self.github.put_labels_on_issue(self.new_pr['issue_number'],
-                                                            self.new_release.get('labels'))
+                            if self.conf.dry_run:
+                                self.logger.info("I would add labels to issue #" +
+                                                 str(self.new_pr['issue_number']))
```

Comment:
Can we move this to `GitHub.put_labels_on_issue()`?

Reply:
Yes, definitely. I didn't do it earlier because I wanted to save some [bytes](https://github.com/Aniket-Pradhan/release-bot/blob/dry-run/release_bot/github.py#L529). I was being greedy.

Anyways, having the condition in `github.py` makes it more uniform-ish. So, on it!